doc/api/api_changes/2017-30-08_AFV_deprecation_of_axes_collision.rst

```diff
@@ -0,0 +1,15 @@
+Deprecation of axes collision
+-----------------------------
+
+The following is related to the PR #9037 (and incidentally to #9034 too).
+
+Adding an axes instance to a figure by using the same arguments as for
+a previous axes instance currently reuses the earlier instance.  This
+behavior has been deprecated in Matplotlib 2.1. In a future version
+(likely 2.3), a *new* instance will always be created and returned.
```

Comment:
I always prefer not indicating the version where removal actually occurs, or is planned to occur: there will be a new note in the changelog when removal actually occurs anyways, and we may as well keep more flexibility this way.

Reply:
That was the reason for using "likely" before "2.3" 😈... Well, I simply removed the mention of the version.